### PR TITLE
Created adapter blueprint with db name in config environment

### PR DESCRIPTION
**Pull request by mattmarcum**
I saw in the dummy app adapter that you had a TODO to grab the db name from some external source.  I think this is the community's preferred method is sticking this stuff in the config.  I went ahead and created an adapter blueprint to match what's in the readme and to pull in the db name from the config.

### CODE REVIEW

File: blueprints/adapter/files/__root__/__path__/__name__.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { Adapter } from 'ember-pouch/index';
```
Comment (fsmanuel) [inline blueprints/adapter/files/__root__/__path__/__name__.js]: you don't need index: `import { Adapter } from 'ember-pouch';

File: tests/dummy/config/environment.js [hunk 2]
```diff
@@ -16,7 +16,9 @@ module.exports = function(environment) {
     APP: {
       // Here you can pass flags/options to your application instance
       // when it is created
-    }
+    },
+
+    emberpouch: {}
```
Comment (fsmanuel) [inline tests/dummy/config/environment.js]:
```
emberpouch: {
  localDb: 'ember-pouch-test'
}
```

File: blueprints/adapter/files/__root__/__path__/__name__.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import PouchDB from 'pouchdb';
+import config from '<%= dasherizedPackageName %>/config/environment';
+import Ember from 'ember';
+
+const { assert, isEmpty } = Ember;
+
+function createDb() {
+  let localDb = config.emberpouch.localDb;
+
+  assert('emberpouch.localDb must be set', !Ember.isEmpty(localDb));
```
Comment (fsmanuel) [inline blueprints/adapter/files/__root__/__path__/__name__.js]: `!isEmpty(localDb)`

**Comment by fsmanuel** [comment]
:+1:

**Comment by mattmarcum** [comment]
Thanks for the code review, fixed those issues.

**Comment by rsutphin** [comment]
I think this is a good idea. I think it would be more in keeping with Ember style for the config key to be camelized (`emberPouch`). Otherwise I think this is a good idea.

Sorry again for the slow response.

File: blueprints/adapter/files/__root__/__path__/__name__.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  let db = new PouchDB(localDb);
+
+  if (config.emberpouch.remoteDb) {
+      let remoteDb = new PouchDB(config.emberpouch.remoteDb);
+
+      db.sync(remoteDb, {
+        live: true,
+        retry: true
+      });
+  }
```
Comment (rsutphin) [inline blueprints/adapter/files/__root__/__path__/__name__.js]: This block is double-indented (4 spaces instead of 2).

**Comment by mattmarcum** [comment]
Ok updated the key and added a section to the README

**Comment by rsutphin** [comment]
Great, thanks. If you could rebase & squash we'll be ready to merge.

**Comment by mattmarcum** [comment]
closing in favor of https://github.com/nolanlawson/ember-pouch/pull/102

**Pull request closed without merging**